psi4/driver/procrouting/proc.py

```diff
@@ -2514,16 +2514,13 @@ def run_cc_property(name, **kwargs):
     all CC property calculations.
 
     """
-    optstash = p4util.OptionsState(
-        ['WFN'],
-        ['DERTYPE'],
-        ['ONEPDM'],
-        ['PROPERTY'],
-        ['CCLAMBDA', 'R_CONVERGENCE'],
-        ['CCEOM', 'R_CONVERGENCE'],
-        ['CCEOM', 'E_CONVERGENCE'])
+    optstash = p4util.OptionsState(['WFN'], ['DERTYPE'], ['ONEPDM'], ['PROPERTY'], ['CCLAMBDA', 'R_CONVERGENCE'],
+                                   ['CCEOM', 'R_CONVERGENCE'], ['CCEOM', 'E_CONVERGENCE'])
 
-    oneel_properties = ['dipole', 'quadrupole']
+    oneel_properties = [
+        'dipole', 'quadrupole', 'mulliken_charges', 'lowdin_charges', 'mayer_indices', 'wiberg_lowdin_indices',
+        'no_occupations'
+    ]
```

Comment:
You might want to check the other functions but OEProp should carry around its valid list. It would be good to pull from there.

Reply:
Good point. I thought that list would include transition dipole/quadrupole but I should have checked.